share/pulse/examples/dice/dpe/DPE.fst

```diff
@@ -1271,12 +1271,14 @@ fn certify_key (sid:sid_t)
   (crt_len:U32.t)
   (crt:A.larray U8.t (U32.v crt_len))
   (t:G.erased trace { trace_valid_for_certify_key t })
-  requires sid_pts_to trace_ref sid t **
+  requires no_extrude <|
+        sid_pts_to trace_ref sid t **
            (exists* pub_key_repr crt_repr.
               pts_to pub_key pub_key_repr **
               pts_to crt crt_repr)
   returns _:U32.t
-  ensures certify_key_client_perm sid t **
+  ensures no_extrude <|
```

Comment:
here we have an exists*, but it's in the ensures ...what happens if you leave out the noextrude?